STARTERKIT/basic/kaizen.theme

```diff
@@ -1,11 +1,26 @@
 <?php
 
+/**
+ * Returns path to svg sprite.
+ */
+function kaizen_get_svg_sprite_path() {
+  return base_path() . \Drupal::theme()->getActiveTheme()->getPath() . '/dist/svg/sprite.svg';
```

Comment:
are you sure this should point to active theme? probably it related only to kaizen and its child themes

Reply:
created follow up for this -> #85